test/oban/job_test.exs

```diff
@@ -129,6 +129,38 @@ defmodule Oban.JobTest do
       assert Job.new(%{}, worker: Fake, unique: [states: [:random]]).errors[:unique]
       assert Job.new(%{}, worker: Fake, unique: [timestamp: :updated_at]).errors[:unique]
     end
+
+    test "empty keys don't require args or meta in fields" do
+      assert Job.new(%{}, worker: Fake, unique: [keys: [], fields: [:worker]]).valid?
+    end
```

Comment:
Hey @sorenone ! Sorry for not checking this fully locally first, CI should now be green and accept this case.

Reply:
@Geekfish No issue! Thank you!